installer/conf/sudoers

```diff
@@ -31,6 +31,9 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 #run hdinsightmanifestreader.rb
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsoft/omsagent/bin/hdinsightmanifestreader.rb
 
+# Enable nxOMSAuditdResource
+omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh
```

Comment:
per oms agent naming convention the bundle names start with lower case (it is actually all lower case , example omsagent-1.3.0-25.universal.deb.x64.sh ).